sourcetracker/sourcetracker.py

```diff
@@ -843,3 +857,112 @@ def _cli_loo_runner(sample, source_category, alpha1, alpha2, beta, restarts,
     o = open(os.path.join(output_dir, sample + '.txt'), 'w')
     o.writelines(lines)
     o.close()
+
+
+def _gibbs(source_df, sink_df, alpha1, alpha2, beta, restarts,
```

Comment:
Will you set default arguments for the tuning parameters? @lkursell may be able to recommend the best values for these